pkg/machinery/scaffold_test.go

```diff
@@ -490,8 +490,19 @@ var _ Template = fakeTemplate{}
 type fakeTemplate struct {
 	fakeBuilder
 
-	body string
-	err  error
+	body            string
+	err             error
+	parseDelimLeft  string
+	parseDelimRight string
+}
+
+func (f fakeTemplate) SetDelim(left, right string) {
+	f.parseDelimLeft = left
+	f.parseDelimRight = right
+}
+
+func (f fakeTemplate) GetDelim() (string, string) {
+	return f.parseDelimLeft, f.parseDelimRight
 }
 
```

Comment:
@ruanxin that seems great, 
Should we not add here some 1 or 2 use cases using the option to ensure that all work fine? 

WDYT?

Reply:
Hi @camilamacedo86, it's a default behavior of go template, I'm not sure if it's make too much sense to test it, but I added two test cases anyway.